### PR TITLE
Services: Visualization

### PR DESCRIPTION
**Pull request by RobertGemmaJr**
- Adds information about CAVE and it's current status
- Adds information about MML collaboration
- Adds links to new CAVE software (MiddleVR, Unity, Motive, etc)
- Adds list of example CAVE projects
- Removes vr.md file
- Closes issue #47

### CODE REVIEW

**Comment by RobertGemmaJr** [comment]
Is there a way to have the `Example Projects` and `Further Reading` lists be in two columns? I think that will look better. The markdown linter doesn't like inline html though.

**Comment by kmilo9999** [comment]
Looks great !! 
It seems the link to the remote rendering documentation is broken. Also, the two first links of CAVE systems are related to the YURT, a system we no longer support.

**Comment by RobertGemmaJr** [comment]
> Looks great !! It seems the link to the remote rendering documentation is broken. Also, the two first links of CAVE systems are related to the YURT, a system we no longer support.

I'll delete the old links. I think I fixed the link to Oscar too - the ID was different then I thought

**Comment by RobertGemmaJr** [comment]
@mirestrepo How would I link to a different part of the website locally? I'm trying to link to the Oscar part of the website but it's not working

**Review by mirestrepo — CHANGES_REQUESTED** [review]
This looks great! Thanks @RobertGemmaJr!

I have few suggestions but this is pretty close!

**Comment by RobertGemmaJr** [comment]
I did take this picture but it's not showing on the website. It's inside of a `<p>` tag which I imagine is what's breaking it. It takes up space with the content picture isn't actually there

studio_4

File: services/visualization/studio-4.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+title: Studio 4
+icon: virtual-reality
+---
+
+Studio 4 in Brown's [Granoff Center](https://arts.brown.edu) is a flexible production space used for teaching and research. It is a fairly basic, workshop-like space that we share with Brown's [MultiMedia Labs](https://sites.google.com/brown.edu/multimedia-labs-new-site/home). The MML are inclusive, adaptive, and experimental spaces equipped with emerging technology to support creative digital work and inspire a community of makers. It has spaces for video production, podcasting, large format 2D printing, 3D printing, and VR. CCV and MML continue to collaborate on a variety of VR projects for the Brown community, with Studio 4 serving as the home base for visualization research and exploration.
+
+![Studio 4](/images/studio_4.jpg)
```
Comment (mirestrepo) [inline services/visualization/studio-4.md:8]:
```suggestion
![Studio 4](/content/images/studio_4.jpg)
```

New path for images coming when [this](https://github.com/brown-ccv/ccv-website-nuxt/pull/89) gets merged

**Review by mirestrepo — APPROVED** [review]
Looks great!. Just accept the change in the suggested path after the other PR is merged and then we can verify that the image shows up

**Comment by mirestrepo** [comment]
🚀  The picture is there and it looks 🔥

**Pull request merged**